edenai_apis/apis/mistral/mistral_api.py

```
@@ -245,6 +245,87 @@ def llm__chat(
         )
         return response
 
+    async def llm__achat(
+        self,
+        messages: List = [],
```

Ruff (0.12.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)